sources/packages/web/src/components/common/restriction/ViewRestriction.vue

```diff
@@ -1,83 +1,144 @@
 <template>
-  <modal-dialog-base :showDialog="showDialog" @dialogClosed="dialogClosed">
-    <template v-slot:content>
-      <v-container class="temporary-modal">
-        <formio
-          formName="viewRestriction"
-          :data="restrictionData"
-          @loaded="formLoaded"
-          @submitted="submitResolution"
-        ></formio>
-      </v-container>
-    </template>
-    <template v-slot:footer>
-      <check-permission-role :role="allowedRole">
-        <template #="{ notAllowed }">
-          <footer-buttons
-            primaryLabel="Resolve Restriction"
-            :disablePrimaryButton="notAllowed"
-            @primaryClick="resolveRestriction"
-            @secondaryClick="dialogClosed"
-          />
-        </template>
-      </check-permission-role>
-    </template>
-  </modal-dialog-base>
+  <v-form ref="viewRestrictionForm">
+    <!-- TODO min-width has to be confirmed  -->
+    <modal-dialog-base
+      title="View restriction"
+      :showDialog="showDialog"
+      max-width="730"
+      min-width="730"
+    >
+      <template #content>
+        <error-summary :errors="viewRestrictionForm.errors" />
+        <h4 class="font-weight-bold" v-if="!restrictionData.isActive">
+          Restriction information
+        </h4>
+        <content-group>
+          <p class="font-weight-bold">Category</p>
```

Comment:
We have a component for this type of key/value item, right?

Reply:
Just curious. What is that component? I was trying to search that

Reply:
<title-value> is the component